.ci/.docker-images.yml

```diff
@@ -0,0 +1,389 @@
+
+  - name: "picklesdoc"
+    repository: "elastic/observability-robots"
+    build_script: "make build"
+    push_script: "make push"
+    working_directory: "apps/pickles"
+
+  - name: "obs-jenkins-heartbeat"
+    repository: "elastic/observability-robots"
+    prepare_script: |
+      pip3 install pyyaml
+      python3 ./generate_heartbeat_configs.py
+    working_directory: "apps/beats/heartbeat"
+
+  # observability-dev Docker Images
+
+  - <<: *observability-dev-docker-images
+    name: "bandstand"
+    working_directory: "apps/automation/bandstand"
+
+  - <<: *observability-dev-docker-images
+    name: "slack-bridge-hey-apm"
+    working_directory: "tools/report-bridge"
+
+  - <<: *observability-dev-docker-images
+    name: "flakey"
+    working_directory: "apps/automation/jenkins-toolbox"
+
+  - <<: *observability-dev-docker-images
+    name: "flakeyv2"
+    working_directory: "apps/automation/flaky-test-analyzer"
+
+  - <<: *observability-dev-docker-images
+    name: "build-analyzer"
+    working_directory: "apps/automation/build-analyzer"
+
+  - <<: *observability-dev-docker-images
+    name: "rebuild-analyzer"
+    working_directory: "apps/automation/rebuild-analyzer"
+
+  - <<: *observability-dev-docker-images
+    name: "integrations-test-reporter"
+    working_directory: "apps/automation/integrations/reporter"
+
+  - <<: *observability-dev-docker-images
+    name: "apm-proxy"
+    working_directory: "tools/apm_proxy/frontend"
+    prepare_script: "git clone https://github.com/haproxytech/spoa-mirror.git"
+
+  - <<: *observability-dev-docker-images
+    name: "apm-proxy-be"
+    working_directory: "tools/apm_proxy/backend"
+
+  # APM ITs Docker Images are built daily.
+
+  - name: "apm-integration-testing"
+    repository: "elastic/apm-integration-testing"
+    tag: "daily"
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/apm-integration-testing-all
+#  - name: "apm-integration-testing-all"
+#    repository: "elastic/apm-integration-testing"
+#    build_script: "make -C docker all-tests"
+#    push_script: "make -C docker all-push"
+
+## This is also failing in Jenkins
+##  https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/oracle-instant-client
+#  - name: "oracle-instant-client"
+#    prepare_script: docker pull store/oracle/database-instantclient:12.2.0.1
+#    build_script: docker tag store/oracle/database-instantclient:12.2.0.1 ${REGISTRY}/${PREFIX}/database-instantclient:12.2.0.1
+#    push_script: docker push ${REGISTRY}/${PREFIX}/database-instantclient:12.2.0.1
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/weblogic/
+#  - name: "weblogic"
+#    prepare_script: docker pull store/oracle/weblogic:12.2.1.3-dev
+#    build_script: docker tag store/oracle/weblogic:12.2.1.3-dev ${REGISTRY}/${PREFIX}/weblogic:12.2.1.3-dev
+#    push_script: docker push ${REGISTRY}/${PREFIX}/weblogic:12.2.1.3-dev
```

Comment:
These images are deprecated by the vendor, we use this build to have they cached in our registry, so they are one-shot builds

Reply:
What should we do with this? It's failing on `docker pull`. Are we good to remove it?

Reply:
>It's failing on docker pull

These images are deprecated and deleted by the vendor

>Are we good to remove it?

You can remove them, we have the Docker image in the registry, if we need newer versions we have to find the new registry and versions

Reply:
Removed in https://github.com/elastic/apm-pipeline-library/pull/1986/commits/bf804dcf5f982130ddde9a8df8e2f328d6c71e2d